From cdbcfbdcf3d81a7cde622a376f4dac15f9a3521d Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 2 Feb 2022 14:38:52 +0100
Subject: [PATCH] tegra-defconfig: Disable Tegra WDT when using PMIC WDT

Having both Tegra WDT and PMIC WDT running
causes the device to reboot when using systemd
to kick the watchdog. Let's leave the PMIC
WDT enabled only, see:
https://github.com/OE4T/meta-tegra/issues/891#issuecomment-1027875304

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 arch/arm64/configs/tegra_defconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/configs/tegra_defconfig b/arch/arm64/configs/tegra_defconfig
index 29ad9696c373..6a092c6a9721 100644
--- a/arch/arm64/configs/tegra_defconfig
+++ b/arch/arm64/configs/tegra_defconfig
@@ -699,9 +699,9 @@ CONFIG_USERSPACE_THERM_ALERT=m
 CONFIG_WATCHDOG=y
 CONFIG_WATCHDOG_NOWAYOUT=y
 CONFIG_MAX77620_WATCHDOG=y
-CONFIG_TEGRA21X_WATCHDOG=y
-CONFIG_TEGRA18X_WATCHDOG=y
-CONFIG_SOFT_PLATFORM_WATCHDOG=y
+#CONFIG_TEGRA21X_WATCHDOG=y
+#CONFIG_TEGRA18X_WATCHDOG=y
+#CONFIG_SOFT_PLATFORM_WATCHDOG=y
 CONFIG_MFD_MAX77620=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
-- 
2.17.1

